.docker/solr/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM solr:7.7
```

Comment:
I'm not sure we need a separate Dockerfile for solr.  I believe we can handle this with docker-compose. e.g.: https://github.com/sul-dlss/argo/blob/master/docker-compose.yml#L50-L56

Reply:
I'm using 7 because that's what were' using in travis.

The separate docker file is in order to create the collection (not sure how to do that from just the docker-compose step).

Reply:
The lines I linked above show how to do this.

Reply:
OK, I'll update that.